flight/Libraries/sanitycheck.c

```diff
@@ -101,11 +101,6 @@ int32_t configuration_check()
 
 	for(uint32_t i = 0; i < num_modes; i++) {
 		switch(modes[i]) {
-			case MANUALCONTROLSETTINGS_FLIGHTMODEPOSITION_MANUAL:
```

Comment:
I think we should keep manual, but we also need to keep manual off any axis of a multirotor.

Right now the net effect of this change is to make things less safe-- people can select "manual" thinking it means "rate" on a quadcopter and the alarm that prevents arming is no longer present.